### PR TITLE
New package: gimp-resynthesizer-2.0.3

### PR DESCRIPTION
**Pull request by mvlahos**
@maxice8 I think I have followed through with your suggestions. If not please let me know and I will do so. Thanks for the review :)
Adds some new filters to gimp for context sensitive removal.

### CODE REVIEW

File: srcpkgs/gimp-resynthesizer/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+revision=0
+wrksrc="resynthesizer-${version}"
+build_style=gnu-configure
+hostmakedepends="automake intltool pkg-config"
+makedepends="gimp-devel glib-devel"
+depends="gimp-python"
+short_desc="Suite of gimp plugins for texture synthesis"
+maintainer="CoolOhm <micvlas@gmail.com>"
+license="GPL-2, GPL-3"
+
```
Comment (maxice8) [inline srcpkgs/gimp-resynthesizer/template]: extra blank line

File: srcpkgs/gimp-resynthesizer/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'gimp-resynthesizer'
+pkgname=gimp-resynthesizer
```
Comment (maxice8) [inline srcpkgs/gimp-resynthesizer/template]: can't the name of upstream be used ?

File: srcpkgs/gimp-resynthesizer/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+version=2.0.3
+revision=0
+wrksrc="resynthesizer-${version}"
+build_style=gnu-configure
+hostmakedepends="automake intltool pkg-config"
+makedepends="gimp-devel glib-devel"
+depends="gimp-python"
+short_desc="Suite of gimp plugins for texture synthesis"
+maintainer="CoolOhm <micvlas@gmail.com>"
+license="GPL-2, GPL-3"
```
Comment (maxice8) [inline srcpkgs/gimp-resynthesizer/template]: use SPDX short-identifier

**Comment by mvlahos** [comment]
this should be ok to merge now.

File: srcpkgs/resynthesizer/template [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'resynthesizer'
+pkgname=resynthesizer
+version=2.0.3
+revision=1
+wrksrc="${pkgname}-${version}"
```
Comment (Johnnynator) [inline srcpkgs/resynthesizer/template]: This is the default value of `wrksrc` you don't have to add it manually.

Comment (CoolOhm) [reply srcpkgs/resynthesizer/template]: Gotcha, I'll change it. Is there a way to update the PR without causing a rebuild?

**Pull request merged**